doc/proposals/scorecard-security.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Testing the Security of Operators by tooling
+authors:
+  - "@camilamacedo"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2019-10-31
+last-updated: 2019-10-31
+status: provisional
+see-also:
+  - "doc/proposals/openshift-4.3/operator-testing-tool.md" 
+---
+
+# Testing the Security of Operators by tooling
+
+## Release Signoff Checklist
+
+- \[ X\] Enhancement is `provisional`
+- \[ X\] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created in [operator-sdk/doc][operator-sdk-doc]
+
+## Open Questions (optional)
+
+- Would have more security checks which may are important to the Operators and are not mapped here? 
+
+## Summary
+
+Currently, the scorecard will perform some tests. For further information see its [doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/scorecard.md#tests-performed).  
+This proposal is regarded to add a new suite of tests in order to ensure the quality of the operators by checking if they are complying with few common/general security practices. 
+In order to clarifies and exemplify this need as solution see the project https://kubesec.io/ and its demo which verifies and check if the resource is complying with the security common practices.   
+
+Note that the checks of this proposal are relevant to any solution which will be deployed in K8s platforms, however, it could be improved in the future in order to accommodate specific implementations and/or checks per type of platform or as suggested to perform further checks in the images used.     
+
```

Comment:
Hi @dmesser and @jmccormick2001, 

I know that we have bigger priorities regards scorecard planned already. However, wdyt over this proposal be merged in the docs here and to be planned to get done after that? Are you both ok with these stories and requirements?